### PR TITLE
Update change-package-repository.md

### PR DESCRIPTION
**Pull request by jiamiao442**
remove vv1.29 to v1.29, and comment exclude

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md [hunk 1]
```diff
@@ -181,19 +181,19 @@ version.
    enabled=1
    gpgcheck=1
    gpgkey=https://pkgs.k8s.io/core:/stable:/v{{< skew currentVersionAddMinor -1 "." >}}/rpm/repodata/repomd.xml.key
-   exclude=kubelet kubeadm kubectl cri-tools kubernetes-cni
+   #exclude=kubelet kubeadm kubectl cri-tools kubernetes-cni
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md:184]: Why this change?

File: content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md [hunk 2]
```diff
@@ -181,19 +181,19 @@ version.
...
    [kubernetes]
    name=Kubernetes
-   baseurl=https://pkgs.k8s.io/core:/stable:/v{{< param "version" >}}/rpm/
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
    enabled=1
    gpgcheck=1
-   gpgkey=https://pkgs.k8s.io/core:/stable:/v{{< param "version" >}}/rpm/repodata/repomd.xml.key
-   exclude=kubelet kubeadm kubectl cri-tools kubernetes-cni
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   #exclude=kubelet kubeadm kubectl cri-tools kubernetes-cni
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md:196]: Why this change?

**Comment by kbhawkey** [comment]
👋 @jiamiao442 
Can you address the comments? It's not clear why you have commented out certain commands.

**Comment by natalisucks** [comment]
@jiamiao442 Hi there – do you have the bandwidth to look over the feedback left by our reviewers? If not, it is perfectly okay to close this pull request and come back to it when you do. Thanks!

**Comment by niranjandarshann** [comment]
I'm going to close this issue. Due to lack of  response on the reviewer comment, It seems that you are not working anymore on it . Feel free to re-open if you disagree, or if you have another path you'd like to try.

/close
